plugins/source/aws/resources/services/timestream/databases.go

```diff
@@ -46,10 +46,11 @@ func Databases() *schema.Table {
 
 func fetchTimestreamDatabases(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- any) error {
 	cl := meta.(*client.Client)
-	svc := cl.Services(client.AWSServiceTimestreamwrite).Timestreamwrite
+	services := cl.Services(client.AWSServiceTimestreamwrite)
+	svc := services.Timestreamwrite
 	// This should be removed once https://github.com/aws/aws-sdk-go-v2/issues/2163 is fixed
-	if cl.AWSConfig != nil && cl.AWSConfig.Region != cl.Region {
-		awsCfg := cl.AWSConfig.Copy()
+	if len(services.AWSConfig.Region) > 0 && services.AWSConfig.Region != cl.Region {
```

Comment:
I think the better fix here might be to set the `Region` in the config in the test helper

Reply:
2e69feb